scss/mixins/typo.scss

```
@@ -0,0 +1,122 @@
+@mixin fontInter {
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+}
+
+@mixin fontSen {
+    font-family: "Sen", Arial, Helvetica, sans-serif;
+}
+
+@mixin headingBig($color: $darkBg) {
+    @include fontSen();
+    color: $color;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: -2px;
+}
+
+@mixin heading1($color: $darkBg) {
```

Chỗ này em có thể tối ưu hơn nữa bằng cách viết một cái mixin fontStyle cho các thuộc tính của font rồi @include vào, nó sẽ gọn hơn rất nhiều, áp dụng tương tự những thằng bên dưới
```suggestion
    @include fontSen;
    @include fontStyle($fontStyle: normal, $fontWeight: 700, $fontSize: 5.6rem, $lineHeight6.4rem, $letterSpacing: 0.2rem, $textColor: $color)
```